### PR TITLE
feat: add source url as a data attribute

### LINKED ISSUE #514: Add source url to markup
When providing a url to the service, some magic (oEmbed call) is used to transform that url into a player snippet for the corresponding service.
When reading the returned markup: there is no way to know the source url. 

Assuming our current ESI include issues will be fixed, this will be the next problem for importing the content from the HTML markup. To solve and assuming we'll get rid of this service after the Blog content will be "re-imported" and transformed into a block based content (vs "inline" and sections at the moment), I propose to add the source url as a data attribute of the parent div in the embed snippet. 
Like this, it is easy to find via a dom query and transform into the appropriate table.

@MarquiseRosier @tripodsan @trieloff WDYT ?

### PR DESCRIPTION
**Pull request by kptdobe**
Fix #514

### CODE REVIEW

File: src/embed.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function toHTML({
   // there is a provider preference, let's go with it.
   if (oEmbed.html) {
     const html = decorateHTML(oEmbed.html, title);
-    return `<div class="embed embed-oembed ${kind}" data-url="${fallbackURL}">
+    return `<div class="embed embed-oembed ${kind}" data-url="${encodeURI(fallbackURL)}">
```
Comment (tripodsan) [inline src/embed.js:75]: no, this is wrong. it needs to be a HTML iso escape (the one with the `&amp;`)

**Review by tripodsan — CHANGES_REQUESTED** [review]
wrong escape

File: src/embed.js (hunk 1 above)
Comment (kptdobe) [reply src/embed.js]: Of course...

File: package.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
     "@adobe/helix-universal": "1.6.7",
     "@adobe/helix-universal-logger": "1.0.7",
     "@braintree/sanitize-url": "5.0.2",
+    "html-entities": "^2.3.2",
```
Comment (tripodsan) [inline package.json]:
```suggestion
    "html-entities": "2.3.2",
```

**Review by tripodsan — APPROVED** [review]

**Pull request merged**